books/free-programming-books-langs.md

```diff
@@ -1949,6 +1949,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Python 3 Official Documentation](https://docs.python.org/3/download.html) (PDF, EPUB, HTML, TEXT) (3.x)
 * [Python 3 Tutorial](https://github.com/Akuli/python-tutorial)
 * [Python Data Science Handbook](https://jakevdp.github.io/PythonDataScienceHandbook) - Jake VanderPlas (HTML, Jupyter Notebooks)
+* [Python For Data Science Cheat Sheet](https://www.utc.fr/~jlaforet/Suppl/python-cheatsheets.pdf) - Data Camp (PDF)
```

Comment:
Please move it to the cheatsheets file [here](https://github.com/EbookFoundation/free-programming-books/blob/main/more/free-programming-cheatsheets.md)

Reply:
I can't edit on this pull request, so I made a new one with the change. I'll close this pull request.